config/configclientauth/README.md

```diff
@@ -0,0 +1,54 @@
+# Client Authorization Settings
```

Comment:
```suggestion
# Authentication Settings for Exporters
```

Reply:
Correct me if I'm wrong, but from the client's perspective, this is only about authentication, not authorization.

Reply:
@jpkrohling this is interesting - yes, it is indeed client authentication but this is done inorder to get a token used for authorization to access resources on the server - not sure how to frame this.